extensions/resteasy-classic/rest-client/pom.xml

```diff
@@ -16,5 +16,6 @@
     <modules>
         <module>deployment</module>
         <module>runtime</module>
+        <module>config</module>
```

Comment:
Why is this a separate module? It doesn't follow our usual model for extensions.

Reply:
@radcortez I want to use the configuration classes in both the classic and reactive rest client extensions, so I need some module that can be consumed by both. I don't see any suitable module that would be already shared by the two client extensions.

Should I create a `resteasy-classic/rest-client-config/runtime/` structure, or something else entirely? I don't want to change the current directory structure too much...